packages/flutter_tools/lib/src/ios/xcode_debug.dart

```diff
@@ -269,8 +269,19 @@ class XcodeDebug {
 
   @visibleForTesting
   XcodeAutomationScriptResponse? parseScriptResponse(String results) {
+    // Some users reported text before the json. Trim any text before the opening
+    // curly brace.
+    // Example: `start process_extensions{"status":true,"errorMessage":null,"debugResult":{"completed":false,"status":"running","errorMessage":null}}`
+    final String trimmedResults;
+    final int jsonBeginIndex = results.indexOf('{');
```

Comment:
Is it possible to have a `{` in the reported text before the JSON? If so, this will still cause decoding to fail. If we're concerned that this is a possibility, we might want to put this into a loop that trims the results further if we fail to decode.

Reply:
I highly doubt there will be a `{` before the JSON. The results here are a response from a `osascript` command. I can add the loop, though, if you think that's better